lp-lsp/README.md

```diff
@@ -17,9 +17,11 @@ The resulting binary will be in `../_build/install/default/bin/lp-lsp`
 
 There is an extension for VSCode derived from VSCoq. To install do:
 
-$ cd editors/vscode/
-$ npm install
-$ ln -s `pwd` ~/.vscode/extensions/
+```
```

Comment:
We could do even better, and use bash syntax coloring (see the main `README.md` for an example).